canvas_modules/common-canvas/src/common-properties/actions/button/button.jsx

```diff
@@ -32,6 +33,35 @@ class ButtonAction extends React.Component {
 		this.uuid = uuid4();
 	}
 
+	getActionButtonKind() {
+		if (!has(this.props, "action.button.kind")) {
+			return CARBON_BUTTON_KIND.TERTIARY;
+		}
+		switch (this.props.action.button.kind) {
+		case CARBON_BUTTON_KIND.PRIMARY: return CARBON_BUTTON_KIND.PRIMARY;
+		case CARBON_BUTTON_KIND.SECONDARY: return CARBON_BUTTON_KIND.SECONDARY;
+		case CARBON_BUTTON_KIND.TERTIARY: return CARBON_BUTTON_KIND.TERTIARY;
+		case CARBON_BUTTON_KIND.GHOST: return CARBON_BUTTON_KIND.GHOST;
+		case CARBON_BUTTON_KIND.DANGER: return CARBON_BUTTON_KIND.DANGER;
+		case CARBON_BUTTON_KIND.DANGER_TERTIARY: return CARBON_BUTTON_KIND.DANGER_TERTIARY;
+		case CARBON_BUTTON_KIND.DANGER_GHOST: return CARBON_BUTTON_KIND.DANGER_GHOST;
+		default: return CARBON_BUTTON_KIND.TERTIARY;
+		}
+	}
+
+	getActionButtonSize() {
+		if (!has(this.props, "action.button.size")) {
+			return CARBON_BUTTON_SIZE.SMALL;
+		}
+		switch (this.props.action.button.size) {
+		case CARBON_BUTTON_SIZE.SMALL: return CARBON_BUTTON_SIZE.SMALL;
+		case CARBON_BUTTON_SIZE.MEDIUM: return CARBON_BUTTON_SIZE.DEFAULT; // TODO: update this after upgrading to carbon 11.x
+		case CARBON_BUTTON_SIZE.LARGE: return CARBON_BUTTON_SIZE.LARGE;
+		case CARBON_BUTTON_SIZE.EXTRA_LARGE: return CARBON_BUTTON_SIZE.EXTRA_LARGE;
+		default: return CARBON_BUTTON_SIZE.SMALL;
```

Comment:
Shouldn't this be `medium` to match the default from before?

Reply:
Default size is `small` - https://github.com/elyra-ai/pipeline-schemas/blob/master/common-pipeline/operators/uihints-v3-schema.json#L628

Default `small` is also returned on line 54.

Reply:
Wasn't the default md before though?  Maybe we need to update the schema.